docs/_sass/_main.scss

```diff
@@ -349,34 +371,34 @@ textarea {
     display: flex;
     flex-wrap: nowrap;
     border-radius: 16px;
-    border: 1px solid $color-gray2;
     padding: 28px;
     font-weight: bold;
     cursor: pointer;
     $border-amount-darken: 15%;
     color: $color-dark;
 
-    &:hover {
-        border-color: darken($color-gray2, $border-amount-darken);
-        color: $color-dark;
+    @mixin card-border($border-color, $border-width: 1.5px) {
+        border-color: $border-color;
+        border-width: $border-width;
+        border-style: solid;
+
+        &.hover {
+            border-color: darken($border-color, $border-amount-darken);
+        }
     }
```

Comment:
I'd recommend pulling out this mixin definition to a `_mixins.scss` partial for better organization